package.json

```diff
@@ -1,21 +1,42 @@
 {
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/cfpb/cfgov-refresh.git"
+  },
   "devDependencies": {
+    "browserify": "^9.0.3",
+    "browserify-shim": "^3.8.6",
     "cf-component-demo": "git://github.com/cfpb/cf-component-demo.git#0.8.2",
-    "grunt": "~0.4.4",
+    "grunt": "~0.4.5",
```

Comment:
Everyone needs to run `npm update` to get this change, `npm install` will only install packages that aren't found in 'node_modules'